source/src/molecular-unfolding/cdk/utils/utils.ts

```diff
@@ -187,4 +187,15 @@ export class AddCondition implements IAspect {
       }
     }
   }
-}
\ No newline at end of file
+}
+
+export class AddSSMPolicyToRole implements IAspect {
+  //arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+  visit(node: IConstruct): void {
+    if (node instanceof iam.Role && node.node.path.endsWith('/Ecs-Instance-Role')) {
+      node.addManagedPolicy({
```

Comment:
prefer to using `ManagedPolicy.fromAwsManagedPolicyName('AmazonSSMManagedInstanceCore')` without hardcode partition